test/cli/ui/prompt_test.rb

```diff
@@ -138,7 +138,15 @@ def test_confirm_no_arrow_keys
         # Windows doesn't detect presses on the arrow keys when picking an option, so we don't show the instruction text
         # for them.
         with_os_mock_and_reload(CLI::UI::OS::Windows) do
-          _run('x', 'n') { Prompt.confirm('q') }
+          # $stdin.getch event doesn't get captured using IO.pipe,
+          # work around is to move the _run command into the test to include stubs
```

Comment:
This is a test, so should be running with true `ENV['TEST']`, meaning it'll use `getc`, and this change shouldn't be needed.